src/test/container-features/containerFeaturesOCI.offline.test.ts

```diff
@@ -0,0 +1,46 @@
+import { assert } from 'chai';
+import { getFeatureManifest, getFeatureBlob, getFeatureRef } from '../../spec-configuration/containerFeaturesOCI';
+import { createPlainLog, LogLevel, makeLog } from '../../spec-utils/log';
+
+export const output = makeLog(createPlainLog(text => process.stdout.write(text), () => LogLevel.Trace));
+
+describe('Test OCI', () => {
+
```

Comment:
I wonder if the github action token could somehow be used for an authenticated pull 🤔.  
It would be very nice to test a non-public oci resource as well. (Perhaps for a future as this is just the initial scaffolding).

Reply:
I left authentication as a TODO, I think the idea would be (like the oras client) to use a token if we have and it mantches, if we don't fall back to anonymous and if that fails try to obtain a token.

We should also cache the tokens so that they can be reused.